lib/ex_unit/lib/ex_unit.ex

```diff
@@ -290,6 +292,15 @@ defmodule ExUnit do
     |> put_seed()
     |> put_slowest()
     |> put_max_cases()
+    |> put_max_failures()
+  end
+
+  defp put_max_failures(opts) do
```

Comment:
I think this function is always returning the same `opts`, no matter what. I think it could be removed.